src/hotspot/share/adlc/main.cpp

```diff
@@ -229,6 +229,7 @@ int main(int argc, char *argv[])
   AD.addInclude(AD._CPP_file, "opto/regalloc.hpp");
   AD.addInclude(AD._CPP_file, "opto/regmask.hpp");
   AD.addInclude(AD._CPP_file, "opto/runtime.hpp");
+  AD.addInclude(AD._CPP_file, "runtime/continuation.hpp");
```

Comment:
This seems unrelated to the AIX changes. Is this include needed in general?

Reply:
This change solved a build issue I had earlier in the development process. The build issue seems to have gone in my recent testing with this change removed. Thanks for pointing this out. I'll change the PR shortly.